fearless/Common/Validation/Validators/BaseDataValidatorFactory.swift

```diff
@@ -119,7 +117,8 @@ extension BaseDataValidatingFactoryProtocol {
 
             if
                 let totalAmount = totalAmount,
-                let minimumBalance = minimumBalance {
+                let minimumBalance = minimumBalance,
+                totalAmount > spendingAmount {
```

Comment:
minimumBalance can be negative number? Why we should check it additional to main check?

Reply:
we are working with unsigned there so it cannot be negative. in case the result of (totalAmount - spendingAmount) will be negative - the app will be crashed. and because minimumBalance cannot be negative, we can say that 'if totalAmount < spendingAmount' that validation cannot be passed by it's logic.

Reply:
got it